src/nrniv/nrncore_write/callbacks/nrncore_callbacks.cpp

```diff
@@ -523,6 +550,37 @@ int core2nrn_corepointer_mech(int tid, int type, int icnt, int dcnt, int* iArray
     return 1;
 }
 
+// NMODL RANDOM seq34 data return from coreneuron
+int core2nrn_nmodlrandom(int tid,
+                         int type,
+                         std::vector<int>& indices,
+                         std::vector<double>& nmodlrandom) {
```

Comment:
```suggestion
                         const std::vector<int>& indices,
                         const std::vector<double>& nmodlrandom) {
```
I believe these should be `const` references.